pkgs/applications/graphics/blockbench-electron/default.nix

```diff
@@ -0,0 +1,52 @@
+{ lib, stdenv, fetchurl, appimageTools, makeWrapper, electron_8 }:
+
+let
+  electron = electron_8;
+in
```

Comment:
This makes the version not easily overwritable. Either use electron_8 directly or inherited it in top-level.